easybuild/easyconfigs/b/Blender/Blender-2.66a-foss-2016.04-trimmed.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'Blender'
+version = '2.66a'
+versionsuffix = '-trimmed'
```

Comment:
@AFCJamie I would drop this suffix personally; I feel it's OK to initially have a slim build of Blender, and then extend it gradually by updating this easyconfig file by adding dependencies etc.